tests/test-vector-ops.js

```diff
@@ -1,93 +1,83 @@
+            deepEqual(VectorOps.getAngle([-1, 0]), { cos: -1, sin: 0 });
 
             // point on the positive y-axis
-            deepEqual(VectorOps.getAngle([0,1]), {"cos" : 0, "sin" : 1});
+            deepEqual(VectorOps.getAngle([0, 1]), { cos: 0, sin: 1 });
 
             // point on the negative y-axis
-            deepEqual(VectorOps.getAngle([0,-1]), {"cos" : 0, "sin" : -1});
+            deepEqual(VectorOps.getAngle([0, -1]), { cos: 0, sin: -1 });
 
             // arbitrary point
             var angle = VectorOps.getAngle([1, 1]);
             ok(Math.abs(angle.cos - Math.sqrt(2) / 2 < 1.0e-15));
             ok(Math.abs(angle.sin - Math.sqrt(2) / 2 < 1.0e-15));
 
             // arbitrary point
-            angle = VectorOps.getAngle([-5*Math.sqrt(3) / 2, -5 / 2]);
-            ok(Math.abs(angle.cos - (-1*Math.sqrt(3))/2) < 1.0e-15);
-            ok(Math.abs(angle.sin - (-1/2)) < 1.0e-15);
+            angle = VectorOps.getAngle([(-5 * Math.sqrt(3)) / 2, -5 / 2]);
+            ok(Math.abs(angle.cos - (-1 * Math.sqrt(3)) / 2) < 1.0e-15);
+            ok(Math.abs(angle.sin + 1 / 2) < 1.0e-15);
         });
 
-        test("Test magnitude", function() {
-            equal(VectorOps.magnitude([0,0]), 0);
-            equal(VectorOps.magnitude([1,1]), Math.sqrt(2));
-            equal(VectorOps.magnitude([1,-1]), Math.sqrt(2));
-            equal(VectorOps.magnitude([-5,-2]), Math.sqrt(29));
+        test("Test magnitude", function () {
+            equal(VectorOps.magnitude([0, 0]), 0);
+            equal(VectorOps.magnitude([1, 1]), Math.sqrt(2));
+            equal(VectorOps.magnitude([1, -1]), Math.sqrt(2));
+            equal(VectorOps.magnitude([-5, -2]), Math.sqrt(29));
         });
 
-        test("Test rotate", function() {
+        test("Test rotate", function () {
             // rotate [0,1] by 0 or 360 degrees
-            deepEqual(
-                VectorOps.rotate([0,1],
-                {"cos" : 1, "sin" : 0}), [0,1]);
+            deepEqual(VectorOps.rotate([0, 1], { cos: 1, sin: 0 }), [0, 1]);
 
             // rotate [-1, -1] by 90 degress
-            deepEqual(
-                VectorOps.rotate([-1,-1],
-                {"cos" : 0, "sin" : 1}), [1,-1]);
+            deepEqual(VectorOps.rotate([-1, -1], { cos: 0, sin: 1 }), [1, -1]);
 
             // rotate [1,1] by 240 degrees
             var angle = {
-                "cos" : (-1/2),
-                "sin" : (-1*Math.sqrt(3)/2)
-            }
-            var rPoint = VectorOps.rotate([1,1],angle);
-            ok(Math.abs(rPoint[0]- 0.3660254037844386) < 1.0e-15);
-            ok(Math.abs(rPoint[1] - -1.3660254037844386) < 1.0e-15);
+                cos: -1 / 2,
+                sin: (-1 * Math.sqrt(3)) / 2,
+            };
+            var rPoint = VectorOps.rotate([1, 1], angle);
+            ok(Math.abs(rPoint[0] - 0.3660254037844386) < 1.0e-15);
+            ok(Math.abs(rPoint[1] + 1.3660254037844386) < 1.0e-15);
```

Comment:
```suggestion
            ok(Math.abs(rPoint[1] - (-1.3660254037844386)) < 1.0e-15);
```

Again not a big issue